Dockerfile

```diff
@@ -17,6 +17,9 @@ RUN CGO_ENABLED=0 GO111MODULE=on go build -a -o network-node-manager main.go
 # Build image
 FROM alpine:3.13.3
 RUN apk add --no-cache iptables=1.8.6-r0 ip6tables=1.8.6-r0
+COPY build/iptables-wrapper-installer.sh /
+RUN chmod 0744 /iptables-wrapper-installer.sh
+RUN /iptables-wrapper-installer.sh
```

Comment:
@mazocode Cloud you merge two RUN commands? such as RUN chmod 0744 /iptables-wrapper-installer.sh && /iptables-wrapper-installer.sh

Reply:
changed.